src/shared/components/oncoprint/controls/CustomDropdown.tsx

```diff
@@ -22,10 +23,10 @@ class CustomButton extends React.Component<any,{}> { // cant type this more spec
 class CustomMenu extends React.Component<any,{}> {
 
     render() {
-        const { children } = this.props;
+        const { className, children } = this.props;
```

Comment:
i'm not sure why the original dev saved these to local variables.  There's no reason and in general it's better just to reference the property on the props object b/c it's clearer (e.g. this.props.className.   This is not a big deal but since you're new, just want to make sure you know.

Reply:
I think it was probably me bc I wrote this class... I think probably it was legacy from when I was trying to extract the rest props in order to pass them through, but then abandoned that or something. anyway you're right